generate/generate.go

```diff
@@ -594,11 +606,18 @@ func writeGeneratedCode(target string, code string) error {
 }
 
 func checkFileCanBeOverwritten(path string) error {
```

Comment:
non-blocking (just a question):  wouldn't it make sense to return a bool from `checkFileCanBeOverwritten` instead of an error?